src/v/cluster/raft0_utils.h

```diff
@@ -25,16 +27,21 @@ static ss::future<consensus_ptr> create_raft0(
   ss::sharded<partition_manager>& pm,
   ss::sharded<shard_table>& st,
   const ss::sstring& data_directory,
-  std::vector<model::broker> initial_brokers) {
+  const std::vector<model::broker>& initial_brokers) {
```

Comment:
nit: could just accept a vector of node ids to show that we don't need anything else